### ISSUE TITLE
🚧 🐞 Bug Report: Container responsive width changes are not applied when using a Template widget

### ISSUE THREAD

**Issue by brettredvers** [elementor/elementor#20677]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When I create a template using containers - and set this up for responsive - the responsive width settings are not applied.

### Steps to reproduce

1) Create a section template and add a couple of containers at 50% width, side by side.
2) Then on mobile change the container widths to 100% (stacked).
3) Add the template to a page with the template widget.
4) View on mobile

The mobile width change to 100% is not applied - it stays at 50%

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.54 (Debian)
	MySQL version: mariadb.org binary distribution v10.6.3
	PHP Version: 8.1.11
	PHP Memory Limit: 512M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 512M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://brackleybrows.clockworkdev.co.uk
	Home URL: https://brackleybrows.clockworkdev.co.uk
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-GB
	Timezone: Europe/London
	Admin Email: jamie@clockwork.co.uk
	Debug Mode: Inactive

== Theme ==
	Name: Brackley Brows
	Version: 1.0
	Author: Clockwork
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.6.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36

== Active Plugins ==
	All-in-One WP Migration
		Version: 7.68
		Author: ServMask

	Breeze
		Version: 2.0.14
		Author: Cloudways

	Elementor
		Version: 3.9.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.9.0
		Author: Elementor.com

== Elementor Experiments ==
	Optimised DOM Output: Active
	Improved asset loading: Active
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility improvements: Active
	Additional Custom Breakpoints: Active
	Import Export Template Kit: Active by default
	Hide native WordPress widgets from search results: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Landing pages: Active by default
	Colour Sampler: Active by default
	Favourite Widgets: Active by default
	Admin Top Bar: Active by default
	Lazy Load Background Images: Inactive by default
	Save as Default: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default

== Log ==
	
PHP: showing 8 of 8PHP: 2022-11-30 22:10:34 [notice X 214][/home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php(10): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/plugin.php(153): class type include()array (
  0 => '/home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php',
)
#2: /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php(18): ElementorPro\Plugin -> autoload()
#3: /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/plugin.php(153): class type include()array (
  0 => '/home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php',
)
#4: /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/modules/notes/data/controller.php(243): ElementorPro\Plugin -> autoload()
',
)]
PHP: 2022-11-30 22:11:36 [notice X 1004][/home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/core/utils/collection.php::75] Return type of ElementorPro\Core\Utils\Collection::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-30 22:29:26 [warning X 4][/home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined array key &quot;data&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-30 22:33:10 [warning X 3][/home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-30 23:05:36 [error X 1][/home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::294] Uncaught Error: Call to a member function update_meta() on null in /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:294
Stack trace:
#0 /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(167): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager-&gt;save_conditions()
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager-&gt;ajax_save_theme_template_conditions()
#2 /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func()
#3 /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module-&gt;handle_ajax_request()
#4 /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-includes/class-wp-hook.php(332): WP_Hook-&gt;apply_filters()
#5 /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-includes/plugin.php(517): WP_Hook-&gt;do_action()
#6 /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-admin/admin-ajax.php(188): do_action()
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-30 23:33:06 [notice X 45][/home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-12-07 19:16:07 [notice X 63][/home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/modules/custom-css/module.php::76] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-12-07 19:57:30 [notice X 444][/home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php(10): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/plugin.php(154): class type include()array (
  0 => '/home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php',
)
#2: /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php(18): ElementorPro\Plugin -> autoload()
#3: /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/plugin.php(154): class type include()array (
  0 => '/home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php',
)
#4: /home/852991.cloudwaysapps.com/dfxfyvvzva/public_html/wp-content/plugins/elementor-pro/modules/notes/data/controller.php(243): ElementorPro\Plugin -> autoload()
',
)]

JS: showing 2 of 2JS: 2022-12-01 02:25:10 [error X 1][https://brackleybrows.clockworkdev.co.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.8.1:3:694254] T.getContainer is not a function 
JS: 2022-12-06 16:08:40 [error X 33][https://brackleybrows.clockworkdev.co.uk/wp-admin/load-scripts.php?c=0&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:4:51824] Cannot read properties of undefined (reading \&#039;replace\&#039;) 

Log: showing 10 of 102022-12-07 16:03:28 [info] elementor::elementor_updater Started 
2022-12-07 16:03:28 [info] Elementor/Upgrades - _on_each_version Start  
2022-12-07 16:03:28 [info] Elementor/Upgrades - _on_each_version Finished 
2022-12-07 16:03:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.1',
  'to' => '3.9.0',
)]
2022-12-07 16:03:28 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.1',
  'to' => '3.9.0',
)]
2022-12-07 19:56:30 [info] elementor-pro::elementor_pro_updater Started 
2022-12-07 19:56:30 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-12-07 19:56:30 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-12-07 19:56:30 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.2',
  'to' => '3.9.0',
)]
2022-12-07 19:56:30 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.2',
  'to' => '3.9.0',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Incompatible

== Elementor Pro - Compatibility Tag ==

**Comment by nicholaszein** [comment]
Hello @brettredvers,

any feedback?

**Comment by nicholaszein** [comment]
Hello there @brettredvers, 

#### 🕙 It's been a while since we answered you, and we haven't received any feedback. 

## 🚧 For this reason, we are closing this issue because of no activity.

👉 If you wish to continue this discussion, please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).
Feel free to submit a new issue in accordance with these guidelines. 😉 

Kind regards
